pkg/minikube/machine/drivers/none/none.go

```diff
@@ -46,8 +46,15 @@ func NewDriver(hostName, storePath string) *Driver {
 	}
 }
 
-// PreCreateCheck checks that VBoxManage exists and works
+// PreCreateCheck checks for correct priveledges and dependencies
 func (d *Driver) PreCreateCheck() error {
+	// check that docker is on path
+	_, err = exec.LookPath("docker")
```

Comment:
This is more of a catch so that users who think minikube does not require docker (as the VM version doesn't) have some information that it is required for the `none` driver.